README.md

```diff
@@ -13,61 +13,118 @@
-
-It has native support for **Prometheus** metrics and
-**OpenTelemetry** traces as well as many other formats like StatsD, Jaeger and Zipkin
-through the OpenTelemetry Collector and is **100% PromQL compliant**.
-It's **full SQL** capabilities enable developers to **correlate metrics, traces and also
-business data** to derive new valuable insights not possible when data is siloed in
-different systems.
-
-Built on top of PostgreSQL and [TimescaleDB](https://www.timescale.com/) it inherits
-**rock-solid reliability, native compression up to 90%, continuous aggregates and the
-operational maturity** of a system that is run on millions of instances worldwide.
-
-For Prometheus users, Promscale provides a robust and highly scalable long-term storage
-system that is 100% PromQL compliant.
-✅ Promscale has consistently been one of the only long-term stores for Prometheus data that continue to maintain its top-performance, receiving
-[100% compliance test score](https://promlabs.com/promql-compliance-test-results/2020-12-01/promscale)
-each time (with no cross-cutting concerns) from PromLab's
-[PromQL Compliance Test Suite](https://promlabs.com/promql-compliance-tests/).
-
-For OpenTelemetry, Jaeger and Zipkin users, Promscale provides the ability to run deep analysis
-on traces using SQL while visualizing the distributed traces in Jaeger or Grafana.
-Install our lightweight [opentelemetry-demo](https://github.com/timescale/opentelemetry-demo)
-on your computer to quickly get a feel of what's possible. More details in
-[this blog post](https://www.timescale.com/blog/learn-opentelemetry-tracing-with-this-lightweight-microservices-demo/).
+<img alt="Promscale" src="docs/assets/promscale-logo.png" width="600px">
+
+Promscale is a unified metric and trace observability backend for Prometheus,
+Jaeger and OpenTelemetry built on PostgreSQL and TimescaleDB.
+
+Promscale serves as a robust and 100% PromQL-compliant Prometheus remote storage
+and as a durable and scalable Jaeger storage backend.
+
+Unlike other observability backends, it has a simple
+and easy-to-manage architecture with just two components: the Promscale
+Connector and the Promscale Database.
+
+[Learn more](https://docs.timescale.com/promscale/latest/about-promscale/#promscale-schema-for-metric-data) 
+about Promscale's architecture and how it works.
 
 <img src="docs/assets/promscale-arch.png" alt="Promscale Architecture Diagram" width="800"/>
 
-This repository contains the source code of the Promscale Connector. Promscale also requires that the Promscale Extension
-which lives in [this repository](https://github.com/timescale/promscale_extension) is installed in the TimescaleDB/PostgreSQL
-database. The extension sets up and manages the database schemas and provides performance and SQL query experience improvements.
+## Promscale for Prometheus
+
+Promscale solves two main Prometheus use cases:
+
+* **Efficient long-term trend analysis and capacity planning**<br/>
```

Comment:
Thanks. Capacity planning is essentially analysis of trends, now? (how fast CPU or memory consumption is growing for example).

If you think they are completely independent we can break it up or maybe rephrase.

Reply:
*Capacity planning* doesn't click a button for me. Feels like losing the context.

Reply:
I've rewritten this section a bit and also moved single pane of glass to the top since that's likely the main reason users look for a product like ours and the value prop there is clearer and easier to understand.